services/ui-src/src/verbiage/pages/mcpar/mcpar-review-and-submit.ts

```diff
@@ -24,6 +24,8 @@ export default {
       additionalInfoHeader: "What happens now?",
       additionalInfo:
         "No further action is needed at this point. CMS will reach out if in the case they have any questions.",
+      printPageUrl: "/mcpar/export",
```

Comment:
This needs its own route in `AppRoutes`. At the moment our logic is to redirect all unexpected paths to `/mcpar` - the dashboard
`<Route path="/mcpar/*" element={<Navigate to="/mcpar" />} />`

Reply:
@gmrabian see https://github.com/CMSgov/mdct-mcr/pull/4920

Reply:
ahhh 👍